scripts/install.js

```diff
@@ -109,18 +109,68 @@ function checkAndDownloadBinary() {
       return;
     }
 
-    download(sass.getBinaryUrl(), sass.getBinaryPath(), function(err) {
-      if (err) {
-        console.error(err);
-        return;
-      }
+    var tmpPath = getTempPath(sass.getBinaryName());
+    if (!(sass.hasBinary(tmpPath))) { // download and install
```

Comment:
As I said I've haven't look over the implementation yet. It's entirely likely there are issues that will need to be addressed.